.github/workflows/build.yml

```diff
@@ -225,7 +225,7 @@ jobs:
           KUBECONFIG: /tmp/ci-kubeconfig
         run: |
           API_SERVER=$( echo -n ${{ secrets.API_SERVER }} | base64 -d)
-          oc login --token=${{ secrets.CLUSTER_TOKEN }} --server=${API_SERVER}
+          oc login --token=${{ secrets.CLUSTER_TOKEN }} --server=${API_SERVER} --insecure-skip-tls-verify=true
```

Comment:
We should remove this right? Our server has a valid certificate as far as I know

Reply:
@dperaza4dustbit My bad. Yes OSD cluster will have the certificate, not in our daily cluster. I was facing faliure when testing with daily cluster https://github.com/openshift-helm-charts/sandbox/runs/7899616080?check_suite_focus=true#step:23:13 . Changed it to take the value from set-env output, similarly as we using during oc_login step